src/ucs/stats/stats.c

```diff
@@ -460,6 +465,138 @@ void ucs_stats_node_free(ucs_stats_node_t *node)
     }
 }
 
+static inline uint32_t ucs_stats_class_id_to_aggregate_sum_base_offset_get(
+        const char *class_name, unsigned *class_id,
+        size_t num_counters_in_class)
+{
+    uint32_t offset = 0;
+
+    if (ucs_likely(*class_id != 0)) {
+        offset = g_stats_class_id_to_name[*class_id].aggregate_sum_class_offset;
+    } else {
+        if (g_stats_class_id_to_name_size == MAX_NUM_OF_STATS_CLASSES) {
+            printf("%s:%d Warning: Statistics class_id to name array size too "
+                   "large!\n",
+                   __func__, __LINE__);
+            return 0;
+        }
+
+        /* Initialize entry */
+        snprintf(g_stats_class_id_to_name[g_stats_class_id_to_name_size]
+                         .class_name,
+                 sizeof(g_stats_class_id_to_name[g_stats_class_id_to_name_size]
+                                .class_name),
+                 "%s", class_name);
+        g_stats_class_id_to_name[g_stats_class_id_to_name_size]
+                .aggregate_sum_class_offset =
+                g_stats_class_id_to_name_last_offset;
+
+        /* Initialize return value */
+        offset = g_stats_class_id_to_name_last_offset;
+
+        /* Return class_id */
+        *class_id = g_stats_class_id_to_name_size;
+
+        /* Update global offset */
+        g_stats_class_id_to_name_last_offset += num_counters_in_class;
+
+        g_stats_class_id_to_name_size++;
+    }
+
+    return offset;
+}
+
+/**
+ * Returns the aggregate-sum of all counters from all interfaces for a compact statistics trace.
+ * Used for Score-P UCX plugin to collect a compact trace of the counters.
+ *
+ * @param [in]  node  Pointer to the statistics node.
```

Comment:
please indent (e.g. "node <more spaces here> Pointer to the statistics node.")

Reply:
Fixed.